kin-backup-and-restore/kin-backup-and-restore-ui-tests/src/main/java/kin/backup/and/restore/ui/tests/BackupAndRestoreDummyActivityForUITest.kt

```diff
@@ -0,0 +1,110 @@
+package kin.backup.and.restore.ui.tests
+
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
+import kin.backupandrestore.BackupAndRestoreManager
+import kin.backupandrestore.BackupCallback
+import kin.backupandrestore.RestoreCallback
+import kin.backupandrestore.exception.BackupAndRestoreException
+import kin.sdk.Environment
+import kin.sdk.KinAccount
+import kin.sdk.KinClient
+
+class BackupAndRestoreDummyActivityForUITest : AppCompatActivity(), View.OnClickListener{
```

Comment:
close BackupManager at onDestory, for completeness and making sure the correct flow works as expected.

Reply:
Forgot to do it.
Will add it.